docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -10,102 +10,128 @@ platforms: ['android', 'ios', 'tvos']
+<Collapsible summary="Example: Check for updates manually">
 
-```objc From Objective-C to import Swift headers for expo-updates
-#import "ExpoModulesCore-Swift.h"
-#import "EXUpdatesInterface-Swift.h"
-#import "EXUpdates-Swift.h"
-```
+You can configure your app to check for updates manually by doing the following steps:
+1. Set the `checkAutomatically` configuration value to `ON_ERROR_RECOVERY` or `NEVER` to disable the module's default launch behavior.
+2. Add the following code to check for available updates, download them, and reload:
+
+    ```jsx App.js
+    import { View, Button } from 'react-native';
+    import * as Updates from 'expo-updates';
+
+    function App() {
+      async function onFetchUpdateAsync() {
+        try {
+          const update = await Updates.checkForUpdateAsync();
+
+          if (update.isAvailable) {
+            await Updates.fetchUpdateAsync();
+            await Updates.reloadAsync();
+          }
+        } catch (error) {
+          // You can also add an alert() to see the error message in case of an error when fetching updates.
+          alert(`Error fetching latest Expo update: ${error}`);
+        }
+      }
+
+      return (
+        <View>
+          <Button title="Fetch update" onPress={onFetchUpdateAsync} />
+        </View>
+      );
+    }
+    ```
+
+</Collapsible>
+
+## Testing
+
+Most of the methods and constants in this module can be used or tested only in release builds. In debug builds, the default behavior is to always load the latest JavaScript from a development server. It is possible to [build a debug version of your app with the same updates behavior as a release build](/eas-update/debug-advanced/#debugging-of-native-code-while-loading-the-app-through-expo-updates). Such an app will not open the latest JavaScript from your development server &mdash; it will load published updates just as a release build does. This may be useful for debugging the behavior of your app when it is not connected to a development server.
+
+**To test the content of an update in a development build**, run [`eas update`](/eas-update/introduction) and then browse to the update in your development build. Note that this only simulates what an update will look like in your app, and most of the [Updates API](#api) is unavailable when running in a development build.
+
+**To test updates in a release build**, you can create a [**.apk**](/build-reference/apk) or a [simulator build](/build-reference/simulators), or make a release build locally with `npx expo run:android --variant release` and `npx expo run:ios --configuration Release` (you don't need to submit this build to the store to test). The full [Updates API](#api) is available in a release build.
```

Comment:
I feel we should link to the P[ublish an update guide](http://docs.expo.dev-pr-29275.s3-website-us-east-1.amazonaws.com/eas-update/publish/) (and if required update that guide). It's okay to introduce the conceptual usage info but I don't think the context is clear to the first timer/unfamiliar developer who wants to use this library